docker/app.base.dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM php:7.4-apache AS wordpress_base
+FROM wordpress:5.5-php7.4-apache AS wordpress_base
```

Comment:
This replaces our copied/forked version of the WordPress Docker file and upgrades to version `5.5`